xmodule/library_content_block.py

```diff
@@ -325,7 +325,9 @@ def _v2_update_children_upstream_version(self):
                 # Since after migration, the component in library is in draft state, we want to make sure that sync icon
                 # appears when it is published
                 child.upstream_version = 0
-                child.save()
+                # Save fails with InvalidScopeError due to `_bound_field_data` field containing LmsFieldData which
+                # is ready-only. This was only seen in production instance and not observed in local dev instance.
+                # child.save()
```

Comment:
```suggestion
                # Save fails with InvalidScopeError due to `_bound_field_data` field containing LmsFieldData which
                # is ready-only. This was only seen in production instance and not observed in local dev instance.
                # child.save()
```
Unless you hope to reinstate `child.save()`, there's no reason to leave a comment on dead code. Just remove it.

Reply:
Yeah, makes sense.